### PR TITLE
palindromic numbers

### PR DESCRIPTION
**Pull request by aepaysinger**
0(n)????

### CODE REVIEW

File: code_challenges/code_wars/palindromic_numbers.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        if len(morphed_number) % 2 == 0:
+            second_half = morphed_number[len(morphed_number) // 2 :]
+        else:
+            second_half = morphed_number[len(morphed_number) // 2 + 1 :]
+        if second_half == front_half[::-1]:
+            return str(count) + " " + morphed_number
+        reverse_morph = morphed_number[::-1]
+        morphed_number = int(reverse_morph) + int(morphed_number)
+        morphed_number = str(morphed_number)
+        count += 1
```
Comment (W-Ely) [inline code_challenges/code_wars/palindromic_numbers.py:15]:
```suggestion
    while True:
        reverse_morph = morphed_number[::-1]
        if reverse_morph == morphed_number:
            return str(count) + " " + morphed_number
        morphed_number = str(int(reverse_morph) + int(morphed_number))
        count += 1
```

**Review by W-Ely — APPROVED** [review]

**Comment by aepaysinger** [comment]
because there is one loop

**Comment by W-Ely** [comment]
Okay, what does O(n) mean?

**Comment by aepaysinger** [comment]
linear?

**Comment by W-Ely** [comment]
> linear?

We spoke in person, this is more like either O(1), O(1 - infinity), or O(random) depending on ones thoughts on Big O.

**Review by W-Ely — APPROVED** [review]

**Pull request merged**